woocommerce/woocommerce
# Feature/marketplace suggestions

### Please re-review this PR…
This is a clean new PR, in woocommerce repo, for the code in https://github.com/woocommerce/woocommerce/pull/22712 (custom fork, PR now closed).

The feedback from that PR's reviews has been addressed. Please refer to the above closed PR for any review comments / discussion. 

When this PR is reviewed and approved, we'll use this branch/PR as the feature branch for Marketplace Suggestions. The follow up PRs will be merged into here once they pass review.

Apologies for the confusion – any questions/problems ping me @haszari and I'll sort it out. Thanks for your help!

cc @kloon @mikejolley @justinshreve @kovshenin @timmyc

### All Submissions:


### Changes proposed in this Pull Request:

Feature branch for Marketplace Suggestions. PRs relating to this feature will land here as they get approved.

Adds support for displaying subtle and appropriate in-product suggestion messaging for WooCommerce Marketplace extensions. These messages are driven by data from an API on WooCommerce.com.

### How to test the changes in this Pull Request:

* Checkout this branch and get it set up as the WooCommerce plugin in your development environment.
* Log in as a store administrator and view suggestions in the products admin screen – there are different suggestions displayed in the empty state and within the list of products.
* Test dismissing suggestions; dismissed suggestions should not show again.

This can also be tested on a [jurassic.ninja site](https://married-white-clawed.jurassic.ninja) - ping me if you want access.
